### PR TITLE
Please do not drink cleaning products. They are not good for you.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request
Makes space cleaner toxic to ingest.

## Why It's Good For The Game

Do not drink, inject, or consume cleaning products.

## Changelog
:cl:
balance: Nanotrasen would like to remind you that you should not consume cleaning products in any way, shape, or form.
/:cl:

### CODE REVIEW

**Comment by RaveRadbury** [comment]
I just want to feel clean and shiny inside. :(

**Comment by AdamElTablawy** [comment]
5% to reduce viral infections by one stage if the virus causes acute respiratory distress syndrome

**Comment by Iamgoofball** [comment]
I am not legitimizing drinking bleach by making it have any form of positive effects @AdamElTablawy. Do not drink bleach.

**Comment by RaveRadbury** [comment]
Akshually it's ammonia, the other ingredient in the forbidden cleaning combo.

**Comment by necromanceranne** [comment]
Fair warning, this makes space cleaner, which is fairly easily created with water and ammomia, one of the most lethal poisons without being a toxin in of itself, so it doesn't suffer from the liver purge. In addition, space cleaner is abundantly available roundstart and premixed in a foam grenade in the form of cleaning grenades, which with this change are more lethal than the syndicate EZ Clean grenades tot janitors get.

**Comment by vpellen** [comment]
Here are some Toxin-Damage-Per-Tick ratings:

Bad Food: 0.5
Cyanide: 1.25
Toxin: 1.5
Carpotoxin: 2.0
Amatoxin: 2.5

New Space Cleaner: 5.0

You made one of the most readily available chemicals the single most damaging toxin in the game, twice as damaging as Amatoxin, for the sole purpose of making a "Fuck Drumph" joke.

And they call _me_ Reddit.

**Comment by tmtmtl30** [comment]
hilarious, but insanely, absurdly unbalanced -- don't make me revise my killmixes

**Comment by AdamElTablawy** [comment]
yeah this currently does 5 tox per tick that's a bit much 

1.25 is fine

**Comment by CRITAWAKETS** [comment]
should only work on injest, otherwise space cleaner grenades become an antag-only thing that can get you valided for

**Comment by Pugie** [comment]
People actually complaining about 5 toxin damage a tick when lexorin exists, might as well just buff all the toxin dealing poisons.

**Comment by Pugie** [comment]
People actually complaining cleaner does 5 toxin when lexorin exists so people will only use cleaner ironically in the same way the only toxin dealing chems used are amanitin and coniine. Might as well just buff all the toxoin dealing poisons.

**Comment by Fikou** [comment]
> People actually complaining cleaner does 5 toxin when lexorin exists so people will only use cleaner ironically in the same way the only toxin dealing chems used are amanitin and coniine. Might as well just buff all the toxin dealing poisons.

not funny

**Comment by itseasytosee** [comment]
5 units to crit. This is balance at its finest

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1055,6 +1055,10 @@
 	color = "#A5F0EE" // rgb: 165, 240, 238
 	taste_description = "sourness"
 	reagent_weight = 0.6 //so it sprays further
+	
+/datum/reagent/space_cleaner/on_mob_life(mob/living/carbon/M)
+	M.adjustToxLoss(5)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:1060]:
```suggestion
	M.adjustToxLoss(0.25)
```

Considering this goes undetected from 'smart-purgers' and is made with ammonia and water, probably best to not have it crit in roughly 20 iterations.

**Comment by itseasytosee** [comment]
this sucks considering the many many ways to get a chemical into your bloodstream. Somone cleaning and you forgot to put on your gas mask today like a good little greytider? toxin damage for you.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1060]: I am going to be adjusting it to completely hose your lungs and throat once I can figure out how organ damage works,

**Comment by RaveRadbury** [comment]
@Iamgoofball Space Cleaner is water and ammonia, ammonia isn't bleach.

**Comment by Tlaltecuhtli** [comment]
does spraying people with space cleaner to clean poison them now?

**Comment by BeeSting12** [comment]
Can you nerf the damage on this

**Comment by tmtmtl30** [comment]
you could probably set non-vapor (i think spray bottles are vapor type) transmission methods (assuming spraying people with space cleaner adds it to their bloodstream) to add to a space cleaner-specific "units ingested/injected" variable that limits the amount of units of space cleaner that will, on metabolization, deal damage. (this does not mean 5 units of toxin damage isn't completely absurd)

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1060]: lol

Comment (vpellen) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1060]: I'm starting to suspect that this change might not actually be a good faith attempt to improve the game

**Comment by PKPenguin321** [comment]
I'm addicted to drinking unbelievably deadly liquid thermite (yeeeeeeaaahhhh)

In a brief moment of lucidity I went to the chemist and told them "Do not, under any circumstances, make me more thermite"

I was back there half an hour later, wearing an italian moustache, and I said "Hello gentlemen, I am Mauser McLauligan, a man you have never met before. Give me 10,000u of thermite directly into my mouth."

**Comment by Iamgoofball** [comment]
made it do 0.25 tox while I figure this out

**Comment by Tharcoonvagh** [comment]
make space cleaner great again

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -1055,6 +1055,10 @@
 	color = "#A5F0EE" // rgb: 165, 240, 238
 	taste_description = "sourness"
 	reagent_weight = 0.6 //so it sprays further
+	
+/datum/reagent/space_cleaner/on_mob_life(mob/living/carbon/M)
+	M.adjustToxLoss(0.25)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:1060]: ingest-only please, or else space cleaner foam and space cleaner sprays will deal minor amounts of tox damage to people

Comment (XDTM) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1060]: given the reference it should also be on inject

Comment (ATH1909) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1060]: ah, fair point

[ingest or inject]-only, then, please

**Comment by ThePainkiller** [comment]
Epic, now I can make 50u space cleaner pills and name them "GreyTide Pods"

**Review by actioninja — CHANGES_REQUESTED** [review]
make it injest or inject only to avoid people poisoning whole rooms with space cleaner nades

**Comment by optimumtact** [comment]
thog don't care

**Pull request closed without merging**